packages/pluggableWidgets/activity-indicator-native/tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
     "extends": "@mendix/pluggable-widgets-tools/configs/tsconfig.base",
     "baseUrl": "./",
-    "include": ["./src"]
+    "include": ["node_modules/@mendix/pluggable-widgets-tools/typings/svg.d.ts", "./src"]
```

Comment:
Can we place this in the `tsconfig.base` that this file inherits from?

Reply:
No, that was what I did at first. `include` will then override `include` from `tsconfig.base`. There is no way of merging two `include` from different tsconfig's

Reply:
Can you make it less specific: `node_modules/@mendix/pluggable-widgets-tools/typings`?